### PR TITLE
No Subtitles Matroska

### LINKED ISSUE #658: Bash script to remove subtitles from Matroska files in a directory
**Is your feature request related to a problem? Please describe.**
We tend to download video files. One of such container for video files is matroska or mkv. Sometimes the subtitles present in those mkv containers would be out of sync and it might be in language which we don't know. So, it would be cool to remove those subtitles and have a clean mkv file with just audio and video.

**Describe the solution you'd like**
A single bash script to traverse through the mkv files, remove the subtitles and meta tags and then place the clean mkv files in a separate folder.

**Describe alternatives you've considered**
 I considered using the MKVToolNixGUI, but when it come to batch removal, we have to manually add and remove each mkv file present in a directory. It was time consuming.

### PR DESCRIPTION
**Pull request by KamalDGRT**
# No Subtitles Matroska

Bash script to remove subtitles from matroska (mkv) files present in a directory.

Fixes #658 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Bash/Multiplexing/No_Subtitles_MKV/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+## Output
+
+#### *Executing the command*
+
+After adding a permanent alias, this command has been executed to get the following output.
+
+- The `VideosDirectory` had `mkv` files. 
+- After the command is executed, a `nosubsmkv` folder will be created with mkv files. 
+
+![Command Execution](https://github.com/KamalDGRT/static/blob/master/RottenScripts/img/03_nosubsmkv_command.png)
```
Comment (HarshCasper) [inline Bash/Multiplexing/No_Subtitles_MKV/README.md:22]: Can you please upload the Images on Imgur and reference them here?

File: Bash/Multiplexing/No_Subtitles_MKV/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+#### *Track listing of the original files*
+
+Let's see the various tracks available in the matroska files.
+
+![Checking](https://github.com/KamalDGRT/static/blob/master/RottenScripts/img/04_trackinfo_original.png)
```
Comment (HarshCasper) [inline Bash/Multiplexing/No_Subtitles_MKV/README.md:28]: Can you please upload the Images on Imgur and reference them here?

File: Bash/Multiplexing/No_Subtitles_MKV/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+#### *Track listing of the new files*
+
+Let's see the various tracks available in the new matroska files.
+
+![Checking](https://github.com/KamalDGRT/static/blob/master/RottenScripts/img/05_trackinfo_new.png)
```
Comment (HarshCasper) [inline Bash/Multiplexing/No_Subtitles_MKV/README.md:34]: Can you please upload the Images on Imgur and reference them here?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
@KamalDGRT Are you working on the same?

**Comment by KamalDGRT** [comment]
I will make the PR by tomorrow evening!

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**